### DISCUSSION TITLE
QField system requirements

### DISCUSSION THREAD

**Discussion by Paul-Losse** [opengisch/QField#2691] [Q&A]
What are the system requirements (software and hardware) for QField V 2.0.14? I have been able to load v2.0.14 on a recent tablet running Android v11from Google Playstore. On an older tablet running Android 6.0.1, it is only possible to download QField v1.10 from the Play store

**Comment by suricactus** [comment]
@Paul-Losse can you try if you are now able to upgrade to the latest QField 2? There was a known issue that was preventing older hardware/software combinations to get upgraded which now should be fixed.
